### PR TITLE
add h2m guide in zh-CN

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
related to #6173.
The h2m process is a little complex, write a guide seems good.

TODO:

- [x] add guides about unsupported elements

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
@yin1999 Could you help me to complete the content of some unfinished? Because I don't know the details. Or remove something you think is unnecessary.

File: docs/zh-cn/h2m_guide.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  ```
+  de、en-US、es、fr、ja、ko、pl、pt-BR、ru、zh-CN、zh-TW、all
+  ```
+
+  转换时根据你的情况选择即可。
+
+- `your_mode` ：主要是 `dry` 和 `replace`。
+  - `dry`：检测是否有不能转换的元素，并将不能转换的信息列在 Yari 目录下生成的 Markdown 报告文件中。如所有文件都可以转换则不生成任何文件。
+  - `replace` ：将 HTML 文件转换为 Markdown 文件，并删除原有的 HTML 文件。
```
Comment (yin1999) [inline docs/zh-cn/h2m_guide.md:52]: 重命名（git mv），而非删除

File: docs/zh-cn/h2m_guide.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+你将在对应的目录下得到转换后的 Markdown 文件。本次转换会产生两次 commit 记录，第一次是重命名，第二次是内容更新。**请不要使用类似于 `git rebase` 这样的命令合并两次提交**，文档准备就绪后直接提交 Pull Request 即可。
+
+### HTML 元素修改指南
+
+如上所述，h2m 工具可能不能够转换某些元素，我们需要对它们进行一些修改再进行转换。你不需要在转换之前检查这个列表，因为在 `dry` 模式下，会给出不能转换的信息，根据信息处理即可。这里列出只是为了告诉你怎样处理。
+
+- 移除所有的 `<span>` 、`<font>`、`<notranslate>` 、`<cite>`、`<sub>`、`<sup>`标签。
+- 除了 `<pre>` 标签，所有的标签属性中 `class="hidden"` 都应被移除。
+- 移除所有的 `summary` 和 `seoSummary`。
```
Comment (yin1999) [inline docs/zh-cn/h2m_guide.md:84]: 其实应该是移除 **除 pre 标签**，以及三类卡片以外的所有 class 样式

File: docs/zh-cn/h2m_guide.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+- 移除所有的 `style` 、`blockIndicator` 属性。
+- `id` 属性只能出现在标题标签里。
+- 移除其他的不能处理的属性，必要时可以和维护者一起讨论如何处理它。
+- 由于元数据的限制，我们的卡片只能使用这些样式和文字：
+  
+```html
+<div class="note">
+  <p><strong>备注：<strong>...</p>
```
Comment (yin1999) [inline docs/zh-cn/h2m_guide.md:92]: 标签不正确，然后请在封闭标签后添加一个空格（虽然能够正常转换，但无法正常渲染）

File: docs/zh-cn/h2m_guide.md [hunk 4]
```diff
@@ -79,12 +79,13 @@ yarn md h2m web/css/_colon_focus --locale=zh-CN --mode=replace
...
-- 除了 `<pre>` 标签，所有的标签属性中 `class="hidden"` 都应被移除。
-- 移除所有的 `summary` 和 `seoSummary`。
+- 移除所有的 `<notranslate>`、`<cite>`、`<sub>`、`<sup>` 标签。
+- 除了 `<pre>` 标签和卡片以外，所有标签的 `class` 属性都应被移除。
 - 移除所有的 `style` 、`blockIndicator` 属性。
 - `id` 属性只能出现在标题标签里。
-- 移除其他的不能处理的属性，必要时可以和维护者一起讨论如何处理它。
+- `<a>` 标签的 `title` 属性应被移除。
+- 没有提到的情况欢迎和维护者一起讨论，并加入到本文档。
+- 非 ASCII 编码的 Unicode 空格会影响转换，需要将它们替换为 ASCII 编码的空格。
```
Comment (yin1999) [inline docs/zh-cn/h2m_guide.md:88]: 换一下顺序会好些

**Review by yin1999 — APPROVED** [review]
LGTM

**Pull request merged**